pramen-py/README.md

```diff
@@ -144,6 +144,37 @@ pramen-py transformations run TransformationClassName --help
 ```
 where TransformationClassName is the name of the transformation.
 
+## Using as a Library
+Read metastore tables by Pramen-Py API
+```python
+import datetime
+from pyspark.sql import SparkSession
+from pramen_py import MetastoreReader
+from pramen_py.utils.file_system import FileSystemUtils
+
+spark = SparkSession.getOrCreate()
+
+hocon_config = FileSystemUtils(spark) \
+    .load_hocon_config_from_hadoop("uri_or_path_to_file")
+
+metastore = MetastoreReader(spark) \
+    .from_config(hocon_config)
+
+# info_date by default "2022-01-01"
+metastore.set_info_date(datetime.date(2022, 2, 1))
```

Comment:
We don't need to ask people to set an information date. When using it as a library, I'd set today automatically as the default info date by the constructor.